admin_test.go

```diff
@@ -80,19 +135,27 @@ func (s *AdminTestSuit) TestCreateTableAsync(c *C) {
 	for i := 0; i < 10; i++ {
 		tblName := fmt.Sprintf("f_%d", i)
 		b, err := s.cli.TableExists(tblName)
-		c.Assert(b, IsTrue)
 		c.Assert(err, IsNil)
+		c.Assert(b, IsTrue)
 
 		tbl := NewTableNameWithDefaultNS(tblName)
 		err = s.cli.DisableTable(tbl)
-		if err != nil {
-			log.Fatal(err)
-		}
+		c.Assert(err, IsNil)
+
 		err = s.cli.DropTable(tbl)
-		if err != nil {
-			log.Fatal(err)
-		}
+		c.Assert(err, IsNil)
+	}
+}
+
+func (s *AdminTestSuit) TestGetPauseTime(c *C) {
+	for i := range retryPauseTime {
+		c.Assert(getPauseTime(i), Equals, retryPauseTime[i]*defaultRetryWaitMs)
 	}
```

Comment:
I don't think this `for` loop is necessary, because if someone change `getPauseTime` algorithm(e.g. add 500 ms), he will update this again, but it's trivial work.

Of course, I agree below border check is necessary.